Changelog.md

```diff
@@ -11,6 +11,7 @@ Bugfixes:
  * Type Checker: Disallow ``virtual`` for modifiers in libraries.
  * ViewPureChecker: Prevent visibility check on constructors.
  * Type system: Fix internal error on implicit conversion of contract instance to the type of its ``super``.
+ * Type system: Fix named parameters in overloaded function and event calls being matched incorrectly if the order differs from the declaration.
```

Comment:
Is this an "important bug"?

Reply:
Maybe? Named parameters and overloads are not obscure features and this renders them unusable in some pretty basic scenarios. On the other hand it looks like it's been broken since 0.5.7 (84b68006ba765bc9608c8a0bf6e85c5de5260fb7) and only now someone complained.

On the other hand I'd say that severity is low. It just prevents valid code from compiling and does not let you compile invalid code and do sneaky things. Its effect is that either there's a valid overload and compiler won't consider it or there's an overload with parameter types that don't match and the compiler will choose it at first but then run into a `TypeError` at a later stage.

Reply:
If it cannot generate invalid code, it is not an important bug. Thanks for the clarification!